CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [Unreleased](https://github.com/Instabug/Instabug-Flutter/compare/v13.4.0...dev)
+
+### Changed
+
+- Set the minimum Flutter SDK version to v2.10.0. ([#518](https://github.com/Instabug/Instabug-Flutter/pull/518))
+
```

Comment:
@ahmedAlaaInstabug how about _"Update the minimum required Flutter SDK version to v2.10.0."_